config/default.cfg

```
@@ -624,7 +624,7 @@ cfg$gms$c30_rotation_scenario_speed <- "by2050"       # def ="by2050"
 # * ("no_marginal"): Marginal land is completely excluded from crop cultivation
 # * Note: Option "q33_marginal" produces the highest spatial correlation
 # * with observed cropland patterns and is recommended for productive runs.
-cfg$gms$c30_marginal_land <- "all_marginal"   # def = "all_marginal"
+cfg$gms$c30_marginal_land <- "q33_marginal"   # def = "all_marginal"
```

Should the default in the comment also be changed?
```suggestion
cfg$gms$c30_marginal_land <- "q33_marginal"   # def = "q33_marginal"
```

---

fixed